src/main/resources/application.yml

```
@@ -1,3 +1,3 @@
 spring:
   profiles:
-    active: dev
+    active: dev
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)